buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/fixture/ElasticsearchFixturePlugin.groovy

```diff
@@ -72,9 +72,13 @@ class ElasticsearchFixturePlugin implements Plugin<Project> {
             integTestCluster.setting("node.ingest", "true")
             integTestCluster.setting("http.host", "localhost")
             integTestCluster.systemProperty('es.http.cname_in_publish_address', 'true')
-        } else if (majorVersion >= 7) {
+        } else if (majorVersion == 7) {
```

Comment:
Why not do it for 7.x also, since then at least we're not using deprecated functionality?